test_runner/fixtures/neon_fixtures.py

```diff
@@ -1572,7 +1572,7 @@ def stop(self, immediate: bool = False) -> "NeonAttachmentService":
             self.running = False
         return self
 
-    def attach_hook(self, tenant_id: TenantId, pageserver_id: int) -> int:
+    def attach_hook_issue(self, tenant_id: TenantId, pageserver_id: int) -> int:
```

Comment:
nit: I would prefer to keep the name of the client function directly aligned with the name of the API it is calling -- i.e. this would take an Optional[int] rather than having separate functions for the _issue, _drop cases.   (usually having separate functions is purer than having an Optional arg, but in the case of a simple API wrapper I'd generally bias toward keeping the mapping to the API very obvious)